assets/model_monitoring/components/src/model_data_collector_preprocessor/trace_aggregator.py

```diff
@@ -8,13 +8,31 @@
 from datetime import datetime
 from pyspark.sql.functions import collect_list, struct
 from shared_utilities.span_tree_utils import SpanTree, SpanTreeNode
-from model_data_collector_preprocessor.mdc_utils import _filter_df_by_time_window
+from model_data_collector_preprocessor.mdc_utils import (
+    _filter_df_by_time_window,
+    _count_dropped_rows_with_error,
+)
 from model_data_collector_preprocessor.genai_preprocessor_df_schemas import (
     _get_aggregated_trace_log_spark_df_schema,
 )
 from shared_utilities.io_utils import init_spark
 
 
+def _validate_traces_df(aggregated_traces_df: DataFrame) -> DataFrame:
+    """Check specific validations for aggregated trace entries as necessary."""
+    print("Validating aggregated trace logs...")
+    # some PF logs that result in exception won't have output so need to drop traces that don't have output or input.
+    transformed_df = aggregated_traces_df.dropna(subset=["input", "output"])
+
+    _count_dropped_rows_with_error(
+        aggregated_traces_df.count(),
+        transformed_df.count(),
+        additional_error_msg="Additionally, the step that caused issues was validating trace logs input/output."
+        " Double check the stdout and spark executor logs for debug info to find root cause of issue.")
```

Comment:
we print out the invalid rows? otherwise how can user check stdout to debug the root cause?

Reply:
I will add some debug logs for traces that are thrown out. They will be in the executor logs mainly

Reply:
Added more debug logs in aggregation function

Reply:
Edited the debug message